src/test/java/org/opensearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -292,6 +321,49 @@ private void verifyPatch(final boolean sendAdminCert, Header... restAdminHeader)
         addUserWithHash("nagilum", "$2a$12$n5nubfWATfQjSYHiWtUyeOxMIxFInUHOAx8VMmGmxFNPGpaBmeB.m",
                 HttpStatus.SC_CREATED);
 
+        // Add enabled service account then get it
+
+
+        rh.sendAdminCertificate = sendAdminCert;
+        response = rh.executePutRequest(ENDPOINT + "/internalusers/happyServiceLive",
+                ENABLED_SERVICE_ACCOUNT_BODY, restAdminHeader);
+        Assert.assertEquals(response.getBody(), HttpStatus.SC_CREATED, response.getStatusCode());
+        rh.sendAdminCertificate = sendAdminCert;
```

Comment:
Probly can remove this re-assignment and the ones below as they are redundant because we are not changing reference for `rh`

Reply:
Will do